### PR TITLE
Fixes #654 - vATIS Profile to include EGPO

### LINKED ISSUE #654: vATIS Profile to include EGPO
## What is the current situation?
EGPO exists however no runway data  is included

## What is the proposed change?
Add EGPO data to include SCO_N SCO_E and SCO_CTR top down cover

## Reference materials (e.g. screenshots)
Scottish vMats

### PR DESCRIPTION
**Pull request by Trodaiche**
Fixes #654

# Summary of changes

Updated as per issue

# Screenshots (if necessary)

N/A

### CODE REVIEW

**Comment by luke11brown** [comment]
@RedstonePilot Able to take a look at this pr?

**Comment by RedstonePilot** [comment]
Unfortunately I'm away for the next week. Can maybe take a look wc 01/04. I don't have much to do with the packs vAtis but will take a look when I'm free!

**Comment by RedstonePilot** [comment]
Seems to be an excess of data. Assuming you made the change in the vATIS window then used what it spat out?

File: UK/vATIS/UK - Scottish AC.json [hunk 1]
```diff
@@ -2667,34 +2789,34 @@
...
+              "low": 977,
+              "high": 994,
+              "altitude": 75
             },
             {
-              "Low": 995,
-              "High": 1013,
-              "Altitude": 70
+              "low": 995,
+              "high": 1013,
```
Comment (RedstonePilot) [inline UK/vATIS/UK - Scottish AC.json:2808]: Seems to be an overlap here. 1013 is the high point while the next block starts at 1012

File: UK/vATIS/UK - Scottish AC.json [hunk 2]
```diff
@@ -1,107 +1,132 @@
 {
-  "Name": "UK - Scottish AC",
-  "Composites": [
+  "name": "UK - Scottish AC",
+  "id": "b1431b6d-4010-48e4-bd25-3e0ae3a1d5fc",
```
Comment (RedstonePilot) [inline UK/vATIS/UK - Scottish AC.json:3]: The id isn't required as far as I know. Not a big issue but increases the file size.

File: UK/vATIS/UK - Scottish AC.json [hunk 3]
```diff
@@ -1,107 +1,132 @@
...
-        {
-          "Name": "RUNWAY 23 (SCO_CTR)",
-          "AirportConditions": "DEPARTING AIRCRAFT MAKE INITIAL CONTACT WITH SCOTTISH CONTROL ON 135.530",
-          "ArbitraryText": null,
-          "Template": "THIS IS [FACILITY] INFORMATION [ATIS_CODE] AT TIME [OBS_TIME]. AUTOMATIC. RUNWAY 23 IN USE. [TL]. [NOTAMS]. [ARPT_COND]. [WIND]. [VIS]. [RVR]. [PRESENT_WX]. [CLOUDS]. [TEMP]. [DEW]. [PRESSURE]. ACKNOWLEDGE RECEIPT OF INFORMATION [ATIS_CODE] AND ADVISE AIRCRAFT TYPE ON FIRST CONTACT."
+      "presets": [
+        {
+          "id": "62464d4d-16c2-48cf-a0f7-bca0f3c6bd59",
+          "name": "RUNWAY 23 (SCO_CTR)",
+          "airportConditions": "DEPARTING AIRCRAFT MAKE INITIAL CONTACT WITH SCOTTISH CONTROL ON 135.525",
```
Comment (RedstonePilot) [inline UK/vATIS/UK - Scottish AC.json:28]: Is this the right frequency? The old one has 135.530 while this says 135.525 (seems to be the pre .833khz).

File: UK/vATIS/UK - Scottish AC.json [hunk 4]
```diff
@@ -1,107 +1,132 @@
...
+          }
         },
         {
-          "Name": "RUNWAY 05 (SCO_CTR)",
-          "AirportConditions": "DEPARTING AIRCRAFT MAKE INITIAL CONTACT WITH SCOTTISH CONTROL ON 135.530",
-          "ArbitraryText": null,
-          "Template": "THIS IS [FACILITY] INFORMATION [ATIS_CODE] AT TIME [OBS_TIME]. AUTOMATIC. RUNWAY 05 IN USE. [TL]. [NOTAMS]. [ARPT_COND]. [WIND]. [VIS]. [RVR]. [PRESENT_WX]. [CLOUDS]. [TEMP]. [DEW]. [PRESSURE]. ACKNOWLEDGE RECEIPT OF INFORMATION [ATIS_CODE] AND ADVISE AIRCRAFT TYPE ON FIRST CONTACT."
+          "id": "5be87c86-a4e4-4f6a-bc95-ef41a4622325",
+          "name": "RUNWAY 05 (SCO_CTR)",
+          "airportConditions": "DEPARTING AIRCRAFT MAKE INITIAL CONTACT WITH SCOTTISH CONTROL ON 135.525",
```
Comment (RedstonePilot) [inline UK/vATIS/UK - Scottish AC.json:37]: same as before. Old frequency.

File: UK/vATIS/UK - Scottish AC.json [hunk 5]
```diff
@@ -1,107 +1,132 @@
...
+          }
         },
         {
-          "Name": "RUNWAY 23 (SCO_D_CTR)",
-          "AirportConditions": "DEPARTING AIRCRAFT MAKE INITIAL CONTACT WITH SCOTTISH CONTROL ON 135.855",
-          "ArbitraryText": null,
-          "Template": "THIS IS [FACILITY] INFORMATION [ATIS_CODE] AT TIME [OBS_TIME]. AUTOMATIC. RUNWAY 23 IN USE. [TL]. [NOTAMS]. [ARPT_COND]. [WIND]. [VIS]. [RVR]. [PRESENT_WX]. [CLOUDS]. [TEMP]. [DEW]. [PRESSURE]. ACKNOWLEDGE RECEIPT OF INFORMATION [ATIS_CODE] AND ADVISE AIRCRAFT TYPE ON FIRST CONTACT."
+          "id": "404aae90-e929-4f56-9fc0-3b98cae4abe7",
+          "name": "RUNWAY 23 (SCO_D_CTR)",
+          "airportConditions": "DEPARTING AIRCRAFT MAKE INITIAL CONTACT WITH SCOTTISH CONTROL ON 135.85",
```
Comment (RedstonePilot) [inline UK/vATIS/UK - Scottish AC.json:82]: Frequency should be 135.855. You've missed a 5!

**Review by RedstonePilot — CHANGES_REQUESTED** [review]
I'm not going to spend all my time reviewing this given that a lot of the frequencies seem to be wrong. If you can fix those then I'll review the whole thing. I would suggest just adding the EGPO profile in rather than editing the whole file, which seems to have lead to mistakes.

**Comment by Trodaiche** [comment]
This was an export of the original profile with only the EGPO updated. I can't answer for the original file

**Comment by RedstonePilot** [comment]
Did you mean to close the pr? If so I'm more than happy to add the required changes I just don't know who has top down responsibility.

**Pull request closed without merging**